.clang-format

```diff
@@ -0,0 +1,2 @@
+BasedOnStyle: Google
+AllowShortIfStatementsOnASingleLine: true
```

Comment:
I'm surprised this isn't already enabled in the Google style. (Maybe the internal config is—dare I say it—Google _plus_?)

Ahem.

Anyway, though, if you could please add a newline at EOF I'd be grateful. 😁

Reply:
addressed in f023d7ca8114575f0ee94a928fdc4b81b2544d4d